etcdctl/README.md

```diff
@@ -69,6 +71,8 @@ The protobuf encoding of the PUT [RPC response][etcdrpc].
 ./etcdctl get foo
 # foo
 # bar
+./etcdctl put foo --ignore-value # this detaches lease
```

Comment:
I think the RPC description should only say what it does, not suggest how to use it. Examples in the docs are better for that.